CONTRIBUTING.md

```diff
@@ -0,0 +1,280 @@
+- Functions using these keywords in their names should have these expected behaviours:
+    - `get`: The function will return a result without modifying anything (except maybe local variables to obtain some result).
+    - `set`: When "set" is understood as an operation rather than a collection of items, the function is expected to change some variable and is unlikely to return a result.
+    - `load`: The function will read something from a file.
+    - `save`: The function will write/overwrite some file with a given content.
+    - `parse`: The function will read some value from a string.
+
+### Function Parameters and Local Variables
+
+- **Prepend an underscore (`_`) to the names of local variables**, in order to avoid collisions with keywords and instance/struct variables.
+
+**YES:**
+
+```gml
+function relerp(_oldfrom, _oldto, _oldvalue, _newfrom, _newto) {
+    if (_oldto == _oldfrom) {
+        throw "relerp() old range has zero width.";
+    }
+    var _amount = (_oldvalue - _oldfrom) / (_oldto - _oldfrom);
+    return _newfrom + (_newto - _newfrom) * _amount;
+}
+```
+
+**NO:**
+
+```gml
+function relerp(oldfrom, oldto, oldvalue, newfrom, newto) {
+    if (oldto == oldfrom) {
+        throw "relerp() old range has zero width.";
+    }
+    var amount = (oldvalue - oldfrom) / (oldto - oldfrom);
+    return newfrom + (newto - newfrom) * amount;
+}
+```
+
+### Toolbox Script Assets
+
+- **Use `utils_` as the prefix, and PascalCase for the rest of the name.** Example: `utils_CommunityToolboxScriptCategory`
+
+### Other assets
+
+- **Use lowercase+underscore asset type prefixes (e.g. `spr_`, `obj_`, `rm_`), followed by PascalCase for the rest of the name.** Example: `rm_Home`
+
+## Code Style
+
+### Identations
+
+- **Use 4 space characters for an indentation level, without tab characters.**
+
+### Braces
+
+**Use [K&R bracing style](https://en.wikipedia.org/wiki/Indentation_style#K&R_style):**
+
+- Opening code block braces are in the same line as block definition (including functions)
+- Closing braces are in their own line, at the same identation level as the original block-starting expression.
+- Prefer using braces whenever applicable, even for one-liners.
+
+Example:
+
+```gml
+function some_function(_arg1, _arg2, _arg3) {
+    var _some_arg = 123;
+    if (_arg1 == _arg2) {
+        // do some thing
+    } else if (_arg1 == _arg3) {
+        // do other thing
+    } else {
+        // do yet another thing
+    }
+}
+```
+
+- If braces are omitted in an `if` block, putting the condition in the first line and the body of the `if` in the second line.
+
+```gml
+function other_function(_arg1) {
+    if (is_undefined(_arg1))
+        return undefined;
+
+    // do the rest
+}
+```
+
+- If there's an `if`-`else` block, then the braces should be consistently applied or consistently omitted.
+
+**YES:**
+
+```gml
+// OK
```

Comment:
Since there's already a "YES" header, you might as well remove the "OK" and similar comments.

Reply:
Removed.